### PR TITLE
fixes #4 : Feature one

### LINKED ISSUE #4: [ CREATE ] Feature One
Create feature one, as shown below. use a suitable image from Unsplash or similar (ensure its free to use)
add some interesting text or use a generator such as [ Pirate Ispum ](https://pirateipsum.me/) or [ Cupcake Ispum ](http://www.cupcakeipsum.com/)

image

#### IF YOU WISH TO WORK ON THIS, PLEASE CHECK NO ONE HAS CLAIMED IT BELOW, IF NOT CLAIM IT BY COMMENTING

### PR DESCRIPTION
**Pull request by Udaygajavalli**
Added feature one with free vector and demo text

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<<<<<<< HEAD
+=======
```
Comment (abdus) [inline style.css:2]: remove this please

File: style.css [hunk 2]
```diff
@@ -126,6 +128,7 @@
 
 /* End Navbar styles */
 
+>>>>>>> e65f4b6f4140c89b380f7785c962d31bfa6c1f8a
```
Comment (abdus) [inline style.css:131]: remove this

**Review by abdus — CHANGES_REQUESTED** [review]

**Review by abdus — APPROVED** [review]

**Comment by abdus** [comment]
resolve conflicts, please.

**Comment by Udaygajavalli** [comment]
It seems everything is fine but again branch conflict.

**Comment by brittneypostma** [comment]
This no longer shows conflicts, I'm going to pull it in. It does not align with the pdf style wise though and will need to be adjusted in another issue. I have added the assets for the projects section to the images folder now.

**Pull request merged**